### ISSUE TITLE
Linux squad xml links

### ISSUE THREAD

**Issue by mjmfighter** [ins0/armasquads#54]
We lost all squad xml files when we transfered to a linux arma box.  We thought at first (and it might have been) and arma issue with squad xmls so we reported it to them.  Recently we were told that they were working for other servers so I started to do some digging.  I hosted a squad xml locally on a server of mine using all lowercase in the path, and it worked ingame!  Linux does not seem to like a mix of lower and upper case in the path name.  

Short version:
Squad xmls do not seem to always work if the path name contains both upper and lowercase letters on linux based server.

**Comment by mjmfighter** [comment]
Doing more research into this, it doesn't seem to be the upper and lowercase causing the issue.  I have copied and pasted the file generated by your side into an xml file onto my site (links below).  The php returned version does not seem to take effect, but the static xml version does.  I am utterly confused at this point.  

armasquads url:
http://armasquads.com/user/squads/xml/6Bww1WUBsnDyfsDbPGzIl2rIyoAjV8df/squad.xml

my static version:
http://mjmfighter.com/squads/squaD.xml

As you can see... they return the exact same data just mine is static.  I have also created a php returned version on my server and it does not take effect as well.  I feel like this might be a problem with arma instead of the site now... just no idea why or how to fix it.

**Comment by ins0** [comment]
thanks @mjmfighter for your research, this helps alot! i will try to investigate this issue tomorrow, it is a pain that for squad xmls requested by server there is no solution to get any sort of error logs or a reason _why_ a squad xml is not displayed correctly. :disappointed: - thanks for your effort!

**Comment by ins0** [comment]
@mjmfighter just looked at the server response and maybe the issue is produced as the xmls on arma squads are delivered with ``gzip`` compression. i will disable the compression tonight may you can test it after that i will keep you informed here when the server config is changed!

**Comment by mjmfighter** [comment]
Hmm... I wonder if its because my server has the 64 bit version of gzip installed.  We are currently running Debian 7 64 bit on our server.  I would install the gzip:i386 version, but debian is complaining about it being an 'essential' package and dont want to test if it will break anything on our running server.

I will try on a test server of mine later tonight if the gzip:i386 package helps and fixes the error.  
And that issue on the arma feedback tracker is mine :)  Its been a problem for a while for us and just recently did I figure out that it works only part of the time.

**Comment by ins0** [comment]
i disabled the ``gzip`` compression on the server so you might test it now please

**Comment by ins0** [comment]
@mjmfighter could you please test if this issue is still present?
